valhalla/jawn/src/controllers/public/traceloopController.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Controller,
+  Request,
+  Route,
+  Post,
+  Get,
+  Tags,
+  Security,
+} from "tsoa";
+import type { JawnAuthenticatedRequest } from "../../types/request";
+import { OTELTrace, TraceManager } from "../../managers/traceManager";
+
+@Route("v1/trace")
+@Tags("Trace")
+@Security("api_key")
+export class TraceController extends Controller {
+
+  @Get("/")
+  public async healthcheck() {
+    this.setStatus(200);
+    return { status: "ok" };
+  }
```

Comment:
why?

Reply:
oh that was just for debugging. issue with JS ipv6 stuff